salt/_macros/certs.jinja

```diff
@@ -103,9 +103,13 @@
       - sls: crypto
       - x509: {{ key }}
 
-{{ crt }}-bundle:
+{{ bundle or crt + '-bundle' }}:
```

Comment:
It's a little strange here that the name of the managed file block is not a path in all cases. When bundle is available, it is. Despite that, when it is bundle, the 'name' attribute is still specified. Perhaps leave name blank since the name of the block is the filename in that case?

Reply:
This was done for compatability with the legacy code, so as to reduce the number of changes necessary to make this work. If there were going to be a future salt cleanup, it'd be worth making prettier - but there won't be, so I felt like minimizing the change footprint was worth the slightly ugly code. :/